eventually/task/models.py

```diff
@@ -67,16 +100,20 @@ def to_dict(self):
             'status': 1,
             'created_at': 1509540116,
             'updated_at': 1509540116,
+            'event' : 13,
+            'users' : [21, 33]

```

Comment:
maybe it is a good idea to write each dict's field from a new line using `|` char.

Reply:
ok